siatune/mm/tasks/mmtrainbase.py

```diff
@@ -56,52 +54,22 @@ def train_model(
         """
         pass
 
-    def context_aware_run(self,
-                          searched_cfg,
-                          backend='nccl',
-                          **kwargs) -> None:
-        """Gather and refine the information received by users and Ray.tune to
-        execute the objective task.
-
-        Args:
-            searched_cfg (Config): The searched configs.
-            backend (str):
-                The backend for dist training. Defaults to 'nccl'.
-            kwargs (**kwargs): The kwargs.
-        """
-        # set non blocking mode on the nccl backend
-        # https://github.com/pytorch/pytorch/issues/50820
-        if backend == 'nccl' and os.getenv('NCCL_BLOCKING_WAIT') is None:
-            os.environ['NCCL_BLOCKING_WAIT'] = '0'
-        return super().context_aware_run(searched_cfg, **kwargs)
-
-    def create_trainable(
-        self,
-        backend: str = 'nccl',
-        timeout_s: int = 1800,
-    ) -> ray.tune.trainable:
+    def create_trainable(self) -> TorchTrainer:
```

Comment:
```suggestion
    def create_trainable(self, backend='nccl') -> TorchTrainer:
```

Reply:
The `backend` argument is no longer required in `create_trainable ` method.

Reply:
Then, the backend for `TorchConfig` is always fixed to `'gloo'`?

Reply:
The `backend` argument is determined by the `init_process_group` in `run` method.